src/modules/display-cards.js

```diff
@@ -0,0 +1,80 @@
+const row = document.querySelector('.row');
+
+const movieName = (id) => {
+  const movies = ['movie1', 'movie2', 'movie3', 'movie4', 'movie5', 'movie6', 'movie7', 'movie8', 'movie9'];
+  return movies[id];
+};
+
+export const RenderCards = (movie, identifier) => {
+  const grid = document.createElement('div');
+  grid.classList.add('col', 'py-2', 'grid', 'col-4');
```

Comment:
- [ ] consider adding a class to make the page responsive on all media widths to make it's UI better